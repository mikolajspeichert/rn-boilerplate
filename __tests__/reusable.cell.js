import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'
import Cell from '@reusable/Cell'

describe('Reusable Cell', () => {
  it('renders correctly', () => {
    expect(
      renderer.create(<Cell id="cell" title="Cell" onPress={jest.fn()} />)
    ).toMatchSnapshot()
  })
})
