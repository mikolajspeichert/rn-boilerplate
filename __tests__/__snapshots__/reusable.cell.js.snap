// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reusable Cell renders correctly 1`] = `
<View>
  Cell
</View>
`;
